samples/kvsWebRTCClientMaster.c

```diff
@@ -139,23 +139,6 @@ INT32 main(INT32 argc, CHAR* argv[])
     return STATUS_FAILED(retStatus) ? EXIT_FAILURE : EXIT_SUCCESS;
 }
 
-STATUS readFrameFromDisk(PBYTE pFrame, PUINT32 pSize, PCHAR frameFilePath)
-{
-    STATUS retStatus = STATUS_SUCCESS;
-    UINT64 size = 0;
-    CHK_ERR(pSize != NULL, STATUS_NULL_ARG, "[KVS Master] Invalid file size");
-    size = *pSize;
-    // Get the size and read into frame
-    CHK_STATUS(readFile(frameFilePath, TRUE, pFrame, &size));
-CleanUp:
-
-    if (pSize != NULL) {
-        *pSize = (UINT32) size;
-    }
-
-    return retStatus;
-}
-
 PVOID sendVideoPackets(PVOID args)
```

Comment:
Can we also move this to `lib/Media.c`?

Reply:
These are sample specific options customers might want to set up. So moving it to `lib` might not make sense. Any definition which is very specific to customer use case like these - it makes sense to let them be in the sample itself.